### PR TITLE
gh-121023: Improve `_xxtestfuzz/README.rst`

### PR DESCRIPTION
**Pull request by sobolevn**
CC @alex who contibutes to the fuzzer.
CC @hugovk who merged https://github.com/python/cpython/pull/107653

* Issue: gh-121023

### CODE REVIEW

**Comment by hugovk** [comment]
Looks fine to me, @illia-v what do you think?

File: Modules/_xxtestfuzz/README.rst [hunk 1]
```diff
@@ -23,18 +23,20 @@ Add the test name on a new line in ``fuzz_tests.txt``.
 
 In ``fuzzer.c``, add a function to be run::
 
-    int $test_name (const char* data, size_t size) {
+    int $fuzz_test_name (const char* data, size_t size) {
```
Comment (illia-v) [inline Modules/_xxtestfuzz/README.rst]:
```suggestion
    static int $fuzz_test_name(const char* data, size_t size) {
```

to make it look like
https://github.com/python/cpython/blob/f98e5678d83d6df02687388fd3d8b2eb9f45b375/Modules/_xxtestfuzz/fuzzer.c#L23

**Review by illia-v — COMMENTED** [review]
Nice correction
Please check one more improvement to the file

**Review by illia-v — APPROVED** [review]

**Review by hugovk — APPROVED** [review]

**Comment by sobolevn** [comment]
Thank you for your reviews!

**Pull request merged**